### PR TITLE
Add MongoDB version and services information

### PR DESCRIPTION
**Pull request by juliamrch**
Add a short line on SSPL MongoDB to explain the available versions on Clever Cloud, and what to do if a customer wants to deploy a higher version. 

Add a page about managed services using Mongo Ops Manager, for customers who are looking for a similar service.

### CODE REVIEW

**Comment by s0dyy** [comment]
Maybe change the name of the commits?

MongoDB: versions and services information...
Metrics: Slack alert with Grafana...

Is the Administrate section the right one? Perhaps this page would be better placed in the Deploy / Addon / MongoDB section? (by creating a folder, like PostgreSQL or MySQL)

**Comment by juliamrch** [comment]
> Is the Administrate section the right one? Perhaps this page would be better placed in the Deploy / Addon / MongoDB section? (by creating a folder, like PostgreSQL or MySQL)

I agree that would make more sense, especially if we add more doc about MongoDB in the future.

File: administrate/managing-MongoDB.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Can I use [Mongo Ops Manager](https://www.mongodb.com/products/ops-manager) on Clever Cloud?
+
+To be able to use Mongo Ops Manager, you'll need a valid MongoDB Enterprise Advanced subscription and to deploy a [Linux version of their manager solution](https://www.mongodb.com/try/download/ops-manager). If you don't have purchased any license from MongoDB and you are using the Community version, you might be looking for a similar service for your databases.
+
+## Does Clever Cloud provide a similar service?
+
+The features available with Mongo Ops Manager could be relevant if you managed your infrastructure and the maintenance of your MongoDB databases yourself. However, if you use Clever Cloud to access our Mongo add-ons, you are already opting for a managed service, and will therefore will benefit from the following features similar to Mongo Ops Manager:
+
+- **A centralized interface** to access your database settings and perform operations on them.
+- **A monitoring and alert system** that can be configured with Grafana (an example with Slack alerts can be found [here](https://www.clever-cloud.com/fr/blog/fonctionnalites/2021/12/03/alertes-slack-pour-grafana/)).
```
Comment (cnivolle) [inline administrate/managing-MongoDB.md:23]: The English version could be better

**Review by cnivolle — APPROVED** [review]

**Comment by juliamrch** [comment]
If we make a folder we'll need to fix. the URLs right after (on it) : https://gohugo.io/content-management/urls/#aliases

**Comment by juliamrch** [comment]
aliases appears to not be working with our Hugo version. Abort mission. Will set it up manually by a search and replace.

**Pull request closed without merging**